src/purchases.ts

```diff
@@ -740,8 +776,8 @@ export default class Purchases {
   public static async setAutomaticAppleSearchAdsAttributionCollection(
     enabled: boolean
   ): Promise<void> {
-    if (Platform.OS === "ios") {
-      RNPurchases.setAutomaticAppleSearchAdsAttributionCollection(enabled);
+    if (Platform.OS === "ios" && enabled) {
+      RNPurchases.enableAdServicesAttributionTokenCollection();
```

Comment:
I did this here so that we don't have to remove this function. Again, I'm okay removing it if y'all think it's appropriate. After all, the underlying function doesn't exist any longer 😅

Reply:
My main question here is whether it works with this implementation. If it works, I think we can leave it (and mark it as deprecated?). If it doesn't work, I think we should remove it.

Reply:
We made it obsolete in iOS so I think we should remove it here. Also AdServices and AppleSearchAds are different so I don't think we should mix them

Reply:
Sounds good - I'll get this removed 👍